candy-machine/program/src/lib.rs

```diff
@@ -27,10 +27,13 @@ use {
     spl_token::state::Mint,
     std::{cell::RefMut, ops::Deref, str::FromStr},
 };
-anchor_lang::declare_id!("cndy3Z4yapfJBmL3ShUp5exZKqR3z33thTzeNMm2gRZ");
+use solana_program::sysvar::SysvarId;
+anchor_lang::declare_id!("6EbJsELBbYMa9DL828YJijPvmRH5j9Y483yoxZVgh9pM");
```

Comment:
Everything looks good here. Did you want to generate a cool public key (like above?). If not, you're good to check in.